### PR TITLE
Authentication users

### PR DESCRIPTION
**Pull request by Fedia23**
The original task was:
1.Create sing up page with obligatory fields: 1.1. firstName; 1.2. lastName; 1.3. email; 1.4. password; 1.5. confirmPassword; 2. Accept this data in appropriate сontroller's method. 3. Save data, and then return full name (first name + last name), on the greeting page. Example: Hello Peter Parker!

Bonus task:

Add filed validation to your dto that you accept in controller.
Show field errors on sign up page if they are exists.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -88,6 +89,18 @@
...
+		<!-- https://mvnrepository.com/artifact/commons-lang/commons-lang -->
+		<dependency>
+			<groupId>commons-lang</groupId>
+			<artifactId>commons-lang</artifactId>
+			<version>${commons.version}</version>
+		</dependency>
+
+
+		<dependency>
+			<groupId>com.microsoft.sqlserver</groupId>
```
Comment (inventorsoft-academy) [inline pom.xml]: Why do you need it?

File: src/main/java/com/thymeleaf/course/domain/model/entity/User.java [hunk 2]
```diff
@@ -1,33 +1,77 @@
...
 import lombok.experimental.FieldDefaults;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.*;
```
Comment (inventorsoft-academy) [inline src/main/java/com/thymeleaf/course/domain/model/entity/User.java]: Don't import all package. Avoid asterisk.

File: pom.xml (hunk 1 above)
Comment (Fedia23) [reply pom.xml]: I deleted this dependency

File: src/main/java/com/thymeleaf/course/domain/model/entity/User.java (hunk 2 above)
Comment (Fedia23) [reply src/main/java/com/thymeleaf/course/domain/model/entity/User.java]: Ok, done.

**Pull request closed without merging**